backend/compact-connect/lambdas/python/common/cc_common/email_service_client.py

```
@@ -216,3 +216,311 @@ def send_privilege_purchase_email(
             },
         }
         return self._invoke_lambda(payload)
+
+    def send_license_encumbrance_provider_notification_email(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)